pkg/operator/controller/clientca-configmap/controller.go

```diff
@@ -0,0 +1,231 @@
+package clientcaconfigmap
```

Comment:
The Ingress Operator's controller count is growing! At some point it may be beneficial to add some high level docs describing what each controller does (and where they live in the codebase).